python/GangaDirac/Lib/Splitters/OfflineGangaDiracSplitter.py

```diff
@@ -174,10 +174,9 @@ def calculateSiteSEMapping(file_replicas, wanted_common_site, uniqueSE, site_to_
 
 def lookUpLFNReplicas(inputs, allLFNData):
     # Build a useful dictionary and list
-    allLFNs = []
-    LFNdict = {}
+    allLFNs = [_lfn.lfn for _lfn in inputs]
+    LFNdict = dict().fromkeys(allLFNs)
```

Comment:
This could have been simply `LFNdict = dict.fromkeys(allLFNs)` to avoid creating a temporary `dict` object unnecessarily.